bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/ReplicationWorker.java

```diff
@@ -228,8 +228,12 @@ public void run() {
         while (workerRunning) {
             try {
                 rereplicate();
+                if (!rereplicate()) {
```

Comment:
Why are you calling the rereplicate method twice?

Reply:
my mistake,I have clean it,
thanks

Reply:
> Why are you calling the rereplicate method twice?

my mistake,I have clean it,
thanks